### PR TITLE
#1934 - Group Member List Ordering

### PR DESCRIPTION
**Pull request by kodinkat**
- fixes: #1934

### CODE REVIEW

File: dt-groups/groups.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ jQuery(document).ready(function($) {
         m.leader = true
       }
     })
-    post.members = window.lodash.sortBy( post.members, ["leader"])
+    post.members = window.lodash.sortBy( post.members, ["post_title"])
```
Comment (corsacca) [inline dt-groups/groups.js:146]: @kodinkat can it do leader, and then member name?

**Comment by corsacca** [comment]
I'm not sure the order is always working:
image

**Comment by kodinkat** [comment]
> I'm not sure the order is always working:
> image

@corsacca - Yep, it seems to be a caps thing; as uppercase will be sorted before lowercase. I'll explore the API further to try and find a suitable option/workaround; which ignores casing..

**Comment by corsacca** [comment]
Thanks @kodinkat !

**Pull request merged**